benches/bench.rs

```diff
@@ -89,6 +89,78 @@ mod poly_benches {
     }
 }
 
+mod commitment_benches {
+    use super::*;
+    use rand::SeedableRng;
+    use rand_xorshift::XorShiftRng;
+
+    /// Benchmarks [de]ser of a univariate commitment
+    fn roundtrip(c: &mut Criterion) {
+        let mut rng = XorShiftRng::from_seed(RNG_SEED);
+        c.bench_function_over_inputs(
+            "Commitment [de]Serializiation",
```

Comment:
```suggestion
            "Commitment Deserializiation",
```

The `[de]` implies to me that this is benchmarking both serialization and deserialization.